pkg/params/run.go

```diff
@@ -19,7 +19,7 @@ const (
 	StartingPipelineRunText = `Starting Pipelinerun <b>%s</b> in namespace
   <b>%s</b><br><br>You can follow the execution on the [%s](%s) PipelineRun viewer or via
   the command line with :
-	<br><code>%s pac logs -L -n %s %s</code>`
+	<br><code>%s pr logs -n %s %s</code>`
```

Comment:
why we are using `tkn pr logs` instead `tkn pac logs` does `tkn pac logs` don't show logs :thinking:

Reply:
tkn pac logs shows the log using "tkn" cli.... "tkn pac logs" only support repositories logging, not a specific pr, which mean when we have multiple parallel pr the tkn pac logs instruction will show the latest one running, not the actual pr.. since to use tkn pac logs you need to have tkn cli installed we just reference it..

(not an issue showing on opc since it's the same binary)

Reply:
:+1: